templates/base/head.tmpl

```diff
@@ -63,7 +63,7 @@
 	<link rel="preload" href="{{AppSubUrl}}/vendor/assets/font-awesome/css/font-awesome.min.css" as="style" onload="this.rel='stylesheet'">
 	<noscript><link rel="stylesheet" href="{{AppSubUrl}}/vendor/assets/font-awesome/css/font-awesome.min.css"></noscript>
 	<link rel="stylesheet" href="{{AppSubUrl}}/vendor/assets/octicons/octicons.min.css">
-
+	<link rel="stylesheet" href="{{AppSubUrl}}/css/main.css">
```

Comment:
Why new css file? I think these css additions need to be done in less file and built into minimized index.css